dotnet/src/Extensions/PromptTemplates.Handlebars/Helpers/KernelHelpers/KernelFunctionHelpers.cs

```diff
@@ -211,7 +211,7 @@ private static void ProcessPositionalArguments(KernelFunctionMetadata functionMe
         if (resultAsObject is RestApiOperationResponse restApiOperationResponse)
         {
             // Deserialize any JSON content or return the content as a string
-            if (string.Equals(restApiOperationResponse.ContentType, "application/json", StringComparison.OrdinalIgnoreCase))
+            if (restApiOperationResponse.ContentType.IndexOf("application/json", StringComparison.OrdinalIgnoreCase) > 0)
```

Comment:
For some requests, such as 204, there might not be a content type, and the ContentType will return null. There is a PR - https://github.com/microsoft/semantic-kernel/pull/5021 to support these cases.

Reply:
Thanks for the call out! I added a null check